game.py

```diff
@@ -0,0 +1,99 @@
+from collections import defaultdict
+
+from etc.utils import CharSpotInWord, read_words_from_file, get_random_word, is_valid_word
+from etc.consts import MAX_ATTEMPTS, INDEX_NOT_FOUND, WORDS_FILE_NAME
+
+
+class WordleGame:
+    def __init__(self):
+        self._words_list = read_words_from_file(WORDS_FILE_NAME)
```

Comment:
beware that if the word list contains a lot of words (say 200,000), this will take up a lot of memory.